jaxopt/_src/isotonic.py

```diff
@@ -100,15 +111,18 @@ def isotonic_l2_pav(y):
   Returns:
     The solution, an array of the same size as y.
   """
-  if not NUMBA_AVAILABLE:
-    warnings.warn(
-        "Numba could not be imported. Code will run much more slowly."
-        " To install, run 'pip install numba'."
-    )
-  # Define the expected shape & dtype of output.
-  shape_dtype = jax.ShapeDtypeStruct(shape=y.shape, dtype=y.dtype)
-  return jax.pure_callback(_isotonic_l2_pav, shape_dtype, y)
-
+  if increasing is True:
+    sign = -1
+  else:
+    sign = 1
+  y = y * sign
+  sol = _isotonic_l2_pav(y) * sign
+  if y_min is None:
+      y_min = -jnp.inf
+  if y_max is None:
+      y_max = jnp.inf
```

Comment:
You can set the default values directly in the function.

Reply:
Done